blog-app/schema/article.py

```diff
@@ -0,0 +1,53 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional
+
+
+class ArticleItem(BaseModel):
+    article_id: int = Field(title="ブログ記事ID")
+    title: str = Field(title="ブログ記事のタイトル")
+    comment_count: int = Field(title="コメント数")
+
+
+class ArticleListResponse(BaseModel):
+    total: int = Field(title="検索結果の総数")
+    offset: int = Field(title="返却リストのオフセット")
+    count: int = Field(title="返却リストの件数")
+    list: List[ArticleItem] = Field(title="記事の返却リスト")
+
+
+class ArticleResponse(BaseModel):
+    article_id: int = Field(title="ブログ記事ID")
+    title: str = Field(title="ブログ記事のタイトル")
+    content: str = Field(title="ブログ記事の中身")
+
+
+class ArticlePostRequest(BaseModel):
+    title: str = Field(title="ブログ記事のタイトル。1文字以上32文字以下")
```

Comment:
この辺りのschemaにも上限下限、また必須パラメータか否かが設定可能なのでハンドラ側に処理を寄せる必要はありません！

```suggestion
    title: str = Field(..., title="ブログ記事のタイトル。1文字以上32文字以下", min_length=1, max_length=32)
```